pkg/cmd/scan/output/output_test.go

```diff
@@ -304,7 +304,7 @@ func TestGetPrinter(t *testing.T) {
 				Options: map[string]string{
 					"path": tt.path,
 				},
-			}); !reflect.DeepEqual(got, tt.want) {
+			}, false); !reflect.DeepEqual(got, tt.want) {
```

Comment:
I think it should be a part of the test case struct and we should add a case with `true`